packages/mutation-testing-elements/README.md

```diff
@@ -106,6 +119,10 @@ Default: _user preference (OS setting)_
 
 Specify in which theme the report needs to be shown. Possibilities: `'light'` or `'dark'`.
 
+### `themeBackgroundColor` [`string`]
+
+Read-only property with the hex code of the background-color from the current theme. 
+
 ### ⚡ `theme-changed` [`CustomEvent<{ theme: string }>`]
```

Comment:
Maybe we can add `{ themeBackgroundColor: string }` here as well for consistency?

Reply:
Good idea! Added it